docs/static-s3-website.md

```diff
@@ -0,0 +1,78 @@
+## Construct StaticS3Website
+
+This construct creates an S3 bucket and upload any content into it.
+It is a composition of the following CDK constructs:
+    - [AWS S3 Bucket lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3)
+    - [AWS S3 Bucket deployment lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3-deployment)
+
+Why to use it over the CDK constructs?
+    - It has good defaults. e.g., enforcing ssl; restricted access; auto deleting bucket on deleting the stack.
+    - easier interfaces. i.e. less configuration
+    - Unified construct
+
+### Usage
+
+#### Simple S3 static website
+
+```ts
+    // This will create the bucket `my-website-bucket` and upload the files in your local `dist` folder into it
+    new StaticS3Website(stack, 'my-website', {
+        bucketName: 'my-website-bucket',
+        deployments: [{
+            sources: ['./dist'],
+        }],
+    });
+```
+
+#### Complex S3 static website
+
+```ts
+    const staticWebsiteConfig: StaticS3WebsiteProps = {
+        bucketName: 'my-website-bucket',
+        removalPolicy: RemovalPolicy.RETAIN,
+        blockPublicAccess: new BlockPublicAccess({
+            blockPublicAcls: true,
+            ignorePublicAcls: true,
+        }),
+        deployments: [
+            {
+                sources: [Source.asset('./dist', { exclude: ['*.html'] })],
+                cacheControl: [CacheControl.fromString('public,max-age=31536000,immutable')],
```

Comment:
Question: this will make the GET of the files on S3 to have response header "cache-control" set? If so, pls add this comment to the example

Reply:
It will add the cache control metadata to the objects of the bucket that will be uploaded by the sources selection.
In short, yes, it will add the cache-control headers to the files